tabpy-server/server_tests/test_pwd_file.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import unittest
+from argparse import Namespace
+from tempfile import NamedTemporaryFile
+
+from tabpy_server.app.app import TabPyApp
+from tabpy_server.app.ConfigParameters import ConfigParameters
+
+try:
+    from unittest.mock import patch, call
+except ImportError:
+    from mock import patch, call
+
+
+class TestPasswordFile(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(TestPasswordFile, self).__init__(*args, **kwargs)
+        self.config_file = None
+        self.pwd_file = None
+        self.cwd = os.getcwd()
+        self.tabpy_cwd = os.path.join(self.cwd, 'tabpy-server', 'tabpy_server')
+
+    def setUp(self):
+        os.chdir(self.tabpy_cwd)
+        self.config_file = NamedTemporaryFile(mode='w', delete=False)
+        self.config_file.close()
+        self.pwd_file = NamedTemporaryFile(mode='w', delete=False)
+        self.pwd_file.close()
+
+    def tearDown(self):
+        os.chdir(self.cwd)
+        os.remove(self.config_file.name)
+        self.config_file = None
+        os.remove(self.pwd_file.name)
+        self.pwd_file = None
+
+    def _set_file(self, file_name, value):

```

Comment:
If you really wanted to do this probably easier to make a partial like `wopen = partial(open, mode='w')`

Reply:
Not sure what you mean. How the code will be different then?